dask_ml/model_selection/_search.py

```diff
@@ -71,6 +71,12 @@
 
 __all__ = ["GridSearchCV", "RandomizedSearchCV"]
 
+# scikit-learn > 1.0.2 removed _check_param_grid
+if SK_VERSION <= packaging.version.parse("1.0.2"):
+    from sklearn.model_selection._search import _check_param_grid
+else:
+    _check_param_grid = None
+
 if SK_VERSION <= packaging.version.parse("0.21.dev0"):
```

Comment:
Unrelated, but can we remove this check as dask-ml now depends on `scikit-learn>=1.0.0`?